f1-telemetry-display/src/main.rs

```diff
@@ -69,7 +69,7 @@ async fn main() {
             log_builder.with_term_logger(LevelFilter::Info, TerminalMode::Mixed, ColorChoice::Auto);
     }
 
-    log_builder.build().expect("Error initializing loggger.");
+    log_builder.build().expect("Error initializing logger.");
```

Comment:
eurgh driveby refactor of a log line, out of scope, DECLINED

Reply:
bruh